addOns/requester/src/main/java/org/zaproxy/addon/requester/internal/HttpPanelSender.java

```diff
@@ -255,6 +266,36 @@ private JToggleButton getButtonHostHeader() {
         return hostHeader;
     }
 
+    private JButton getButtonLowerCaseHeaderNames() {
+        if (lowerCaseHeaderNames == null) {
+            lowerCaseHeaderNames =
+                    new JButton(ExtensionRequester.createIcon("lowercase-header-button.png"));
+            lowerCaseHeaderNames.setToolTipText(
+                    Constant.messages.getString(
+                            "requester.httpsender.button.lowerCaseHeadersName"));
+            lowerCaseHeaderNames.addActionListener(e -> lowerCaseHeaderNames());
+        }
+        return lowerCaseHeaderNames;
+    }
+
+    private void lowerCaseHeaderNames() {
+        customHttpPanelRequest.saveData();
+        HttpMessage msg = ((HttpMessage) customHttpPanelRequest.getMessage());
+        setLowerCaseHeaderNames(msg);
+        customHttpPanelRequest.updateContent();
```

Comment:
IMO this could be done in the lambda and have the following method called`lowerCaseHeaderNames`.

Reply:
are you telling me to name the method to lowercase the header name as lowerCaseHeaderNames, but we have JButton name's same as lowerCaseHeaderNames , is this not cause any problem ?

Reply:
A method with that name already exists, I'm suggesting using that name for the other method and have this logic in the lambda.